settings.gradle

```diff
@@ -7,5 +7,5 @@ include ':helios-parser'
 include ':helios-benchmarks'
 include ':helios-sample'
 include ':helios-optics'
-include 'helios-dsl-syntax-compiler'
-
+include ':helios-dsl-syntax-compiler'
+include ':helios-retrofit-converter'
```

Comment:
I would name this just `helios-retrofit` so we have a consistent naming convention for integration modules.